README.md

```diff
@@ -29,9 +24,9 @@
 </div>
 
 More relevant links:
-* Talk to us on [Element][b2] or on [Discord][c2] in the [`ink_smart-contracts`](https://discord.com/channels/722223075629727774/765280480609828864) channel
+* Find answers to your questions by joining our [Stack Exchange][b2] community
```

Comment:
Good that you bring this up, I think it requires a bit more discussion on how we want to engage with the community from now on. Not sure if we should remove mentions of talking to us in general. I've just added a talking point for the retreat next week on this topic. Let's wait with merging this until then, ok?